src/release-it-config/README.md

```diff
@@ -0,0 +1,123 @@
+# Silvermine-release
+
+If you have this project installed globally you can use the following commands:
+
+
+## Run a Release
+
+With a clean working directory, run:
+
+`silvermine-release --release`
+
+Begins the release process for a full release:
+
+   * Generates a changelog of bugfixes and features.
+   * Calculates the next semver version number and bumps the version in `package.json`.
+   * Commits the changes.
+
+## Run a Prerelease
+
+With a clean working directory, run:
+
+`silvermine-release --pre-release=rc`
+
+...or an alpha:
+
+`silvermine-release --pre-release=alpha`
+
+Begins the release process for a pre-release version:
+
+   * Calculates the next semver version number and bumps the version in `package.json`.
```

Comment:
ADIRL, we need to update this because it's not automatically calculated for pre-releases. Also, @pbredenberg is going to open an issue to investigate having it do so.

Reply:
As of the latest commits we're getting the correct bump now.